e2e-tests/recover-no-primary/conf/some-name-exposed.yml

```diff
@@ -0,0 +1,55 @@
+apiVersion: psmdb.percona.com/v1
+kind: PerconaServerMongoDB
+metadata:
+  name: some-name
+spec:
+  #platform: openshift
```

Comment:
it might be better to add `clusterServiceDNSMode: "Internal"` to the cr. It makes the changes more obvious. I know that we use this value by default too but we can forget it easily. :)

Reply:
to this CR or to the default CR?

Reply:
in the test at least.